src/components/AdvertiserName/AdvertiserNameToggle.tsx

```diff
@@ -31,11 +31,18 @@ const AdvertiserNameToggle = memo(({ advertiserInfo, onToggle }: TAdvertiserName
                 <Text lineHeight='lg' size='sm'>
                     <Localize i18n_default_text='Show my real name' />
                 </Text>
-                <Text className='advertiser-name-toggle__label-real-name' color='less-prominent' lineHeight='xs'>
+                <Text
+                    className='advertiser-name-toggle__label-real-name'
+                    color='less-prominent'
+                    data-testid='dt_advertiser_label_real_name'
+                    lineHeight='xs'
+                >
                     {advertiserInfo?.fullName}
                 </Text>
             </div>
-            <ToggleSwitch onChange={onToggleShowRealName} value={shouldShowRealName} />
+            <div data-testid='dt_advertiser_name_toggle_switch'>
```

Comment:
why do we need a new div for this? can we pass `data-testid` inside `ToggleSwitch`?

Reply:
Hi @farrah-deriv, I have tried it initially, but data-testid is not showing up in the <togggleSwitch > when passed, hence I had to add a div for it.

Reply:
@farrah-deriv , as per discussion, this div has been removed.